apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -125,8 +135,24 @@ class Downloads extends React.Component {
 
   render() {
     const {platform} = this.state;
+    // const {enableExperiments} = request.params['enableExperiments']
     return (
       <div>
+        <h1>CS Discoveries Maker Toolkit IT Setup</h1>
+        {/*Adding logic here
+        if request.params['enableExperiments'] === 'microbit'
```

Comment:
a couple things here:

1. this file has an example of how to check if an experiment is enabled:

https://github.com/code-dot-org/code-dot-org/blob/fd61236110e5d6be13b03fdcfa635592a0de9f9e/apps/src/lib/kits/maker/ui/SetupGuide.jsx#L56

2. since we're already in JSX here, we can't use if/else to conditionally render. there are a lot of different ways to conditionally render content in react/JSX, so it's probably useful to read through [this documentation about it](https://reactjs.org/docs/conditional-rendering.html)

Reply:
Perfect!
I was not sure at all about how to check experiments. This was one of the main questions I had.
I was aware of the implementation of the conditional rendering, and there are some examples if the same file. The documentation is super useful.

Reply:
Implemented!